### PR TITLE
Add flatpak manifest for org.kde.ktechlab

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

**Comment by ognarb** [comment]
bot, build org.kde.ktechlab

**Comment by ognarb** [comment]
bot, build org.kde.ktechlab

**Comment by nedrichards** [comment]
@flathub/kde

**Comment by Siosm** [comment]
@ognarb Looks good from a quick first look. Can you check if some files need to be excluded via `cleanup`? Thanks

File: org.kde.ktechlab.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+   "id": "org.kde.ktechlab",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15",
+   "sdk": "org.kde.Sdk",
+   "command": "ktechlab",
```
Comment (cho2) [inline org.kde.ktechlab.json:6]: Icon not show. Add `"rename-icon": "ktechlab",` to showing the icon.

File: org.kde.ktechlab.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
```
Comment (cho2) [inline org.kde.ktechlab.json:10]: `Qt: Session management error: Could not open network socket`. 
Add `"--share=network",`

Comment (bilelmoussaoui) [reply org.kde.ktechlab.json:10]: I don't think that's necessary.

Comment (tsdgeos) [reply org.kde.ktechlab.json:10]: It's not necessary, but it's nicer not to have spurious warnings

Comment (bilelmoussaoui) [reply org.kde.ktechlab.json:10]: Which is something the user never sees. Unless the application uses network,  it doesn't need access to it.

Comment (tsdgeos) [reply org.kde.ktechlab.json:10]: The default instructions in flathub.org are to use the command line to install and run.

**Comment by ognarb** [comment]
> @ognarb Looks good from a quick first look. Can you check if some files need to be excluded via `cleanup`? Thanks

Sure will do next week.

**Comment by nedrichards** [comment]
bot, build org.kde.ktechlab

**Comment by barthalion** [comment]
bot, build org.kde.ktechlab

**Comment by barthalion** [comment]
Looks like there's mostly `/include` and `/mkspecs`. The rest is fine.

**Comment by tsdgeos** [comment]
Looks reasonable to me, besides the small cleanup that @barthalion mentions, have you submitted the appdata patch upstream?

**Comment by Mailaender** [comment]
https://invent.kde.org/sdk/ktechlab/-/commit/82927e1c99c37ecffdf4810634b9d471440002b3 Yes, the patch was submitted upstream.

**Comment by travier** [comment]
@Mailaender If you are interested in picking this one up you may open a new PR for this one.

**Comment by Mailaender** [comment]
Was this rejected because it is https://download.kde.org/unstable/ktechlab/ ?

**Comment by hfiguiere** [comment]
it was closed because whoever submitted it didn't update it for a year. PR just get autoclosed.

**Pull request closed without merging**